app/bundles/WebhookBundle/Form/Type/WebhookType.php

```diff
@@ -91,7 +77,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $builder->add(
             'events',
-            'choice',
+            ChoiceType::class,
```

Comment:
You'll have to flip the labels and values for choices and use the `choices_as_values` in this bundle as well. See https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit# for details.

Reply:
Done. TY!